code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -436,4 +436,68 @@
 	back = /obj/item/weapon/storage/backpack/satchel/leather
 	head = /obj/item/clothing/head/HoS/syndicate
 	mask = /obj/item/clothing/mask/cigarette/cigar/havana
-	glasses = /obj/item/clothing/glasses/thermal/eyepatch
\ No newline at end of file
+	glasses = /obj/item/clothing/glasses/thermal/eyepatch
+
+//Ancient cryogenic sleepers. Players become NT crewmen from a hundred year old space station, now on the verge of collapse.
+/obj/effect/mob_spawn/human/oldsec
+	name = "old cryogenics pod"
+	desc = "A humming cryo pod. You can barely recognise a security uniform underneath the built up ice. The machine is attempting to wake up its occupant"
+	mob_name = "a security officer"
+	icon = 'icons/obj/cryogenic2.dmi'
+	icon_state = "sleeper"
+	roundstart = FALSE
+	death = FALSE
+	random = TRUE
+	mob_species = /datum/species/human
+	flavour_text = "<font size=3><b>Y</b></font><b>ou are a security officer working for Nanotrasen, stationed onboard a state of the art research station. You vaguely recall rushing into a \
+	cryogenics pod due to a oncoming radioation storm. The last thing you remember is the station's Artifical Program telling you that you would only be asleep for eight hours. As you open \
```

Comment:
don't forget that you work for Nanotrasen, not NanoTrasen, two whole different companies!